### DISCUSSION TITLE
Apa perbedaan subscription link antara @master dan @latest?

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1421] [Q&A] [answered]
![20230630_161657_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/7123adbc-514c-49ba-99c3-82d9ba698853)

**Comment by realodix** [comment, marked as answer]
Baik `@master` atau `@latest`, keduanya sama saja, tidak ada bedanya.

`@latest` hanyalah nama alias saja

Karena terjadi kebingungan, Saya telah menghapus link `@master` di https://github.com/realodix/AdBlockID/commit/c8ea67b7ebc1d55dce687f91bf8157365bb20788
